ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py

```diff
@@ -1127,3 +1127,33 @@ def test_jax_lax_ge(
         x=xs[0],
         y=xs[1],
     )
+
+
+# cosh
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_jax.valid_float_dtypes),
```

Comment:
Please also add:
`as_variable=st.booleans()`
`native_array=st.booleans()`
These are required inputs for tests. Check the [docs](https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#ivy-tan) for further explanation

Reply:
Hi @Auxia , it don't need to add 
```
as_variable=st.booleans()
native_array=st.booleans()
```
 by adding **@handle_cmd_line_args**, would you like to make a check for it?